Add user tab

PanSpagetka opened the pull request:
Add user tab.

There is no `Last modified` column, because we don't get this information from API. I think we don't need this because we can't edit users.

Filtering is not working.
Bottom pagination line is not present, because we don't have total count yet. This should be updated when API provides us total count.
![Screenshot from 2020-02-07 12-36-32](https://user-images.githubusercontent.com/9535558/74026443-80a64300-49a6-11ea-99d1-512dfaabf954.png)

mmenestr commented:
@PanSpagetka just a few comments:
1. For the header in the first column **"Username" should be one word,** not "User name" with a space in between.
2. Is it possible to add a description? as seen in the mocks?
3. The spacing for the header is a little off
- without a description, the header should have 24px of spacing above and below it
- with a description, the header should have 24px above it, 8px between the header and description, and 24px spacing right underneath it (before the start of the grey part of the page)
can see in more detail here [](https://marvelapp.com/4bf315h/screen/66016993)
(The blue/green rectangle means 24px spacing, and the bright green rectangle means 8px spacing)!

PanSpagetka commented:
@mmenestr I have fixed issues you have mentioned and updated the screenshot. Just one more thing, do we have the URL for `user management list` link?

karelhala commented:
The `user management list` should land users in https://www.qa.redhat.com/wapps/ugc/protected/usermgt/userList.html where `qa` part is only for ci and qa envs.

karelhala commented on src/presentational-components/shared/top-toolbar.js:
```diff
@@ -38,7 +38,7 @@ export const TopToolbarTitle = ({ title, description, children }) => (
         </TextContent>
         { description &&
-          <TextContent className="pf-u-pt-sm pf-u-mb-md">
+          <TextContent className="pf-u-pt-sm pf-u-mb-md top-toolbar-description">
```
Can you change the class to `ins-c-rbac__users-toolbar-description` are the `-` and `__` correct @ryelo?

karelhala commented on src/smart-components/user/users.js:
```diff
@@ -0,0 +1,46 @@
+        <TopToolbar paddingBottm={ false }>
+          <TopToolbarTitle
+            title="Users"
+            description='These are the users in your organization. To add new users to your organization go to user management list.'
```
Double quotes in here.

karelhala commented on src/smart-components/user/users.js:
```diff
@@ -0,0 +1,46 @@
+import UsersList from '../group/add-group/users-list';
+
+const Users = () => {
+  const [ selectedRows, setSelectedRows ] = useState([]);
```
If we are not going to use the state in here, no need to pretend it like this. Rather use `() => undefined` in `setSelectedUsers` prop.

karelhala commented on src/smart-components/user/users.js:
```diff
@@ -0,0 +1,46 @@
+const Users = () => {
+  const [ selectedRows, setSelectedRows ] = useState([]);
+
+  const renderUsersList = () =>
```
No need to assign this to function call, but rather return this directly.

mmenestr commented:
@PanSpagetka yes, like Karel said, this is the link to the user management page https://www.redhat.com/wapps/ugc/protected/usermgt/userList.html

Also, one small thing to fix. The header and the description are not left aligned with each other. The description should be a few pixels to the left to be aligned!
